src/olympia/editors/management/commands/award_post_review_points.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+from django.core.management.base import BaseCommand
+
+import olympia.core.logger
+from olympia import amo
+from olympia.activity.models import ActivityLog
+from olympia.editors.models import ReviewerScore
+
+
+log = olympia.core.logger.getLogger('z.editors.award_post_review_points')
+
+# Note we're going to add to the ReviewerScore this commands adds to be able
+# to find them later.
```

Comment:
somehow this comment reads weird. It's a bit too much "add" and somehow doesn't sound right. Nitpick though

Reply:
` # The note we're going to add to the ReviewerScore this command creates to be able to find them later.`